libxrender will fail pkgconf test pipeline

If the pkgconf test pipeline is added to libxrender it will fail in the following way:

```
2024/11/12 23:12:55 INFO running step "test/pkgconf"
2024/11/12 23:12:55 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:12:55 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:12:55 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:12:55 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:12:55 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:55 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:55 WARN + basename /home/build/melange-out/libxrender-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:55 WARN + dev_pkg=libxrender-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:55 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:55 WARN + apk info -L libxrender-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:55 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:55 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:55 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:55 WARN + grep -q ^Name: usr/lib/pkgconfig/xrender.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:55 WARN + basename usr/lib/pkgconfig/xrender.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:55 WARN + lib_name=xrender uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:55 WARN + echo usr/lib/pkgconfig/xrender.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:55 WARN + grep -q '^usr/lib/pkgconfig/xrender.pc$\|^usr/share/pkgconfig/xrender.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:55 WARN + pkgconf --exists xrender uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:55 WARN + pkgconf --print-errors --exists xrender uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:55 WARN Package xproto was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:55 WARN Perhaps you should add the directory containing `xproto.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:55 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:55 WARN Package 'xproto', required by 'xrender', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:55 WARN Package 'renderproto', required by 'xrender', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:55 WARN Package 'x11', required by 'xrender', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:56 INFO pod 668f5020659e81be502e1a5e5096c85a89f8a09be1fba691893aa479b39f6628 terminated
2024/11/12 23:12:56 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:12:56 INFO   workspace dir: /home/user/tmp/melange-workspace-54424798
2024/11/12 23:12:56 INFO   guest dir: /home/user/tmp/melange-guest-1677781331
2024/11/12 23:12:56 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/libxrender] Error 1
```

--- a/libxrender.yaml
+++ b/libxrender.yaml
@@ -43,8 +43,13 @@ subpackages:
       - uses: split/dev
     dependencies:
       runtime:
+        - libx11-dev
         - libxrender
+        - xorgproto
     description: libxrender dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
   - name: libxrender-doc
     pipeline:

--- a/libxrender.yaml
+++ b/libxrender.yaml
@@ -1,7 +1,7 @@
 package:
   name: libxrender
   version: 0.9.11
-  epoch: 7
+  epoch: 8
   description: X Rendering Extension client library
   copyright:
     - license: XFree86-1.1
@@ -43,9 +43,7 @@ subpackages:
       - uses: split/dev
     dependencies:
       runtime:
-        - libx11-dev
         - libxrender
-        - xorgproto
     description: libxrender dev
     test:
       pipeline:

